### ISSUE TITLE
Push new version?

### ISSUE THREAD

**Issue by Sommerregen** [esheldon/pycolumns#14]
Hi,

I really like your module! :-D :+1: Any chance it gets updated soon to a newer release?

**Comment by esheldon** [comment]
Yes, I could tag a new version and make it available on pypi.  However, note that the dependencies have changed from 1.0

Have you tried the version in master to see if it works for you?

**Comment by Sommerregen** [comment]
Hi @esheldon ,

I installed the latest commit.

 ```
pip install git+https://github.com/esheldon/pycolumns.git@d3c70a88586bdd66581eef5fbb52a4fcf1e3343f#egg=pycolumns
```

and it works. However, I noticed some unfinished work: some debug messages are printed even though `verbose` is turned off, the code example in the readme is not complete, etc. I've looked at the code base and can understand why you haven't released a new version yet. Maybe it is better to release a new version only after a full review of the code and comments on API changes. If anyone wants to test the new version, they can at least find an issue about it and try on their own. Thanks for your work!
